influxdb3/Cargo.toml

```diff
@@ -97,3 +97,4 @@ test_helpers.workspace = true
 tonic.workspace = true
 tower.workspace = true
 test-log.workspace = true
+predicates = "3.1.3"
```

Comment:
What's this? We generally want all our dependencies in the workspace.

Reply:
It's a library used by the `assert_cmd` crate to assert on CLI stdout with code that looks like this (snipped from this PR, there are other examples where the inverse logic is applied with the `.not()` method):
```
    // validate we get a debug message indicating upload disabled
    AssertCmd::cargo_bin("influxdb3")
        .unwrap()
        .args(serve_args)
        .arg("-vv")
        .arg("--disable-telemetry-upload")
        .timeout(std::time::Duration::from_millis(200))
        .assert()
        .failure()
        .stdout(predicates::str::contains(expected_disabled));
```

I added it using `cargo add --dev -p ...` and didn't noticed that it wasn't added to the workspace -- I'll do that now!